app/validators/relative_date_validator.rb

```diff
@@ -2,11 +2,11 @@ class RelativeDateValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     return if value.nil?
 
-    if options[:before].present? && options[:before].call <= value
-      record.errors.add(attribute, :date_before, message: options[:before_message] || "must be before #{options[:before].call}")
+    if options[:before].present? && options[:before].call < value
```

Comment:
@ryanb-gds and I chatted on Slack about potentially refactoring this Relative Date Validator class to make use of [Ranges](https://ruby-doc.org/3.2.2/Range.html).

Just noting this here for visibility – but I don't expect that to be changed in this PR 🙂